docs/dev_guides/changing_splink/blog_posts.md

```diff
@@ -1,8 +1,8 @@
 # Contributing to the Splink Blog
 
-Thanks for considering making a contribution to the [Splink Blog](../../blog/index.md)! We are keen to use this blog as a forum all things data linking and Splink! 
+Thanks for considering making a contribution to the [Splink Blog](../../blog/index.md)! We are keen to use this blog as a forum all things data linking and Splink!
 
-This blog, and the docs as a whole, are build using the fantastic [mkdocs-material](https://squidfunk.github.io/mkdocs-material/), to uderstand more about how the blog works under the hood checkout out the mkdocs-material [blog documentation](https://squidfunk.github.io/mkdocs-material/blog/2022/09/12/blog-support-just-landed/).
+This blog, and the docs as a whole, are built using the fantastic [mkdocs-material](https://squidfunk.github.io/mkdocs-material/), to uderstand more about how the blog works under the hood checkout out the mkdocs-material [blog documentation](https://squidfunk.github.io/mkdocs-material/blog/2022/09/12/blog-support-just-landed/).
```

Comment:
understand

Reply:
I'll run the spellchecker against this (as I should have done previously...) and double check there are no more mistakes.